python/pyspark/sql/snappy/snappysession.py

```diff
@@ -61,9 +61,9 @@ def createTable(self, tableName, provider=None, schema=None, allowExisting=True,
         :return: :class:`DataFrame`
         """
         if provider is None:
-            provider = self.getConf("spark.sql.sources.default", "org.apache.spark.sql.parquet")
+            provider = self.conf.get("spark.sql.sources.default", "org.apache.spark.sql.parquet")
         if schema is None:
-            df = self._jsparkSession.createTable(tableName, provider, allowExisting, options)
+            df = self._jsparkSession.createTable(tableName, provider, options, allowExisting)
         else:
```

Comment:
Good catch. :-) The createTable method should preferably match the signature of the Scala equivalent class https://github.com/SnappyDataInc/snappydata/blob/master/core/src/main/scala/org/apache/spark/sql/SnappySession.scala

Reply:
May be options are kept as the last argument as options are accepted as varargs? Unlike Scala counterpart where options are accepted as map. Not sure whether Scala has syntactic sugar to accept varargs as key-value pairs as the argument list.